languages/elixir/exercises/concept/high-score/.docs/instructions.md

```diff
@@ -1,7 +1,5 @@
 In this exercise, you're implementing a way to keep track of the high scores for the most popular game in your local arcade hall.
 
-You have 7 functions to implement, all related to returning and manipulating a map of high score data.
-
```

Comment:
This is no longer true, it's 6. But I would rather remove it because other exercises don't have those intros either.